src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportMarketoBulkExtractTablesToPL.sql

```diff
@@ -44,9 +44,18 @@ BEGIN TRANSACTION
 /* Delta Code */
 
 /* Delta Update MarketoLeads */
-
+;with baseMarketoLeadsData as
+(
+		Select distinct MLData.LeadId,MLData.[FirstName],MLData.[LastName],MLData.[EmailAddress],MLData.[CreatedAt],MLData.[UpdatedAt],EmpData.EmployerHashedId,ProviderData.Ukprn As ProviderID
+		from stg.MarketoLeads  MLData   LEFT JOIN 
```

Comment:
As Marketo Imports are Delta, If we joining on Staging Table to get Employer HashedId and Provider Id then we will have them only for latest but will miss on the data that is already in PL

Reply:
As discussed, add the update statement for existing data updates to marketo leads table

Reply:
Amend the update statement for EmployerHashedID and ProviderID